lib/internals/handler.ts

```diff
@@ -70,7 +87,8 @@ export function Handler(method?: HttpVerb): MethodDecorator {
         if (err instanceof HttpException) {
```

Comment:
Would call the single exception `message` to not make it too similar to `errors`.

Returning the `message` in the `errors` prop as array if `err.errors` is empty, would make it easier for FE to just use one field when they are implementing error handling.

Proposed solution:

```
const statusCode = err.statusCode ?? 500;
const message = err.message ?? 'An unknown error occurred.';

res.status(statusCode).json({
  statusCode,
  message,
  errors: err.errors ?? [message],
  stack: 'stack' in err && process.env.NODE_ENV === 'development' ? err.stack : undefined
});
```